src/main/java/com/cookingchef/model/Cart.java

```diff
@@ -0,0 +1,123 @@
+package com.cookingchef.model;
+
+import java.util.Optional;
+
+public class Cart {
```

Comment:
Since this class seems to represent an element of the cart, you shouldn't name it `Cart`, it's misleading
```suggestion
public class CartEntry {
```

Reply:
Je suis d'accord pour le Cart et CartEntry ça me genait qu'il y ait pas